source/_components/light.hue.markdown

```diff
@@ -26,11 +26,13 @@ If you want to enable the light component directly, add the following lines to y
 light:
   platform: hue
   host: DEVICE_IP_ADDRESS
+  username: DEVICE_USERNAME
```

Comment:
Since this is a optional argument it should not be in the configuration example. Always try to minimize the configuration example since it's often cut and pasted.